### PR TITLE
move to changelog params

### PR DESCRIPTION
**Pull request by timosachsenberg**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -98,240 +98,6 @@ Cleanup/Removal:
 - Deprecated (and likely removed in the next release)
   - XTandemAdapter
 
-- CHANGED PARAMETERS:
```
Comment (mr-c) [inline CHANGELOG:101]: Please mention `CHANGELOG_PARAMS` here, there are no mentions of it anywhere in the repo

**Review by mr-c — COMMENTED** [review]
FYI, I find this table unreadable. Is this format specific for a particular consumer?

**Comment by timosachsenberg** [comment]
Was intended to simplify (semi-)automated updates via GKN. Currently not used.
So feel free to suggest a better format that works with the different types and adapt the scripts to create them. I think this was a quick shot to "at least have something in place"

File: CHANGELOG (hunk 1 above)
Comment (timosachsenberg) [reply CHANGELOG:101]: good point can do - or you just suggest in-line and I accept

**Comment by mr-c** [comment]
> Was intended to simplify (semi-)automated updates via GKN. Currently not used.

Ah, okay; good to know!

> So feel free to suggest a better format that works with the different types and adapt the scripts to create them. 

-> https://github.com/OpenMS/OpenMS/issues/7537


> I think this was a quick shot to "at least have something in place"

Better than nothing, agreed!

**Pull request merged**